Corona-Warn-App/src/main/res/drawable/tan_input_digit.xml

```diff
@@ -7,12 +7,14 @@
         </shape>
     </item>
     <item
-        android:height="@dimen/submission_tan_input_digit_stroke"
-        android:gravity="bottom">
+        android:top="-4dp"
+        android:right="-4dp"
+        android:left="-4dp">
```

Comment:
what is the reason behind this? I prefer to create a drawable for Lollipop devices specifically if this causing issue on it

Reply:
The width and height attributes are only supported from Android 6 on; see commit message here: https://github.com/corona-warn-app/cwa-app-android/pull/2955/commits/823356a6bf16e30a74e17734f220732090051b09

Agreed, will put into separate file.